core/base/pod_vector.h

```diff
@@ -0,0 +1,164 @@
+/****************************************************************************
```

Comment:
This class is super dangerous to use. Please do not commit this change.
Instead of we should change TrianglesCommand::Triangles to have std::vector inside

Reply:
Please remove this class or at least create many unit tests to prove that it works as expected

Reply:
At least, for some specific use cases, pod_vector faster than std::vector:
```txt
Runing perftest ...
ff######## Test round 1 ######### 
Testing xx_pod_vector...--> count: 49950000000, cost: 0.176476(s), realloc_hits: 0
Testing ax_pod_vector...--> count: 49950000000, cost: 0.103016(s), realloc_hits: 0
Testing std_vector...--> count: 49950000000, cost: 0.141327(s)


######## Test round 2 ######### 
Testing xx_pod_vector...--> count: 49950000000, cost: 0.168548(s), realloc_hits: 0
Testing ax_pod_vector...--> count: 49950000000, cost: 0.105139(s), realloc_hits: 0
Testing std_vector...--> count: 49950000000, cost: 0.140052(s)


######## Test round 3 ######### 
Testing xx_pod_vector...--> count: 49950000000, cost: 0.170028(s), realloc_hits: 0
Testing ax_pod_vector...--> count: 49950000000, cost: 0.103696(s), realloc_hits: 0
Testing std_vector...--> count: 49950000000, cost: 0.140359(s)
```

refer to: https://github.com/yasio/yasio/actions/runs/3213606435/jobs/5253397887